presto-hive/src/test/java/com/facebook/presto/hive/TestHiveDistributedQueriesWithAsyncPageTransport.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive;
+
+import com.facebook.presto.tests.AbstractTestDistributedQueries;
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Optional;
+
+import static com.facebook.presto.hive.HiveQueryRunner.createQueryRunner;
+import static io.airlift.tpch.TpchTable.getTables;
+
+public class TestHiveDistributedQueriesWithAsyncPageTransport
+        extends AbstractTestDistributedQueries
+{
+    public TestHiveDistributedQueriesWithAsyncPageTransport()
+    {
+        super(() -> createQueryRunner(
+                getTables(),
+                ImmutableMap.of(
+                        "exchange.async-page-transport-enabled", "true"),
```

Comment:
Since it is enabled by default, what we need to smoke-test is the old way

Reply:
Make sure you change the class name respectively.

Reply:
> Since it is enabled by default, what we need to smoke-test is the old way

Default will be false for a short term,  I put default as true temporarily for test purpose.